app/assets/javascripts/discourse/app/lib/sidebar/user/categories-section/category-section-link.js

```diff
@@ -171,6 +198,13 @@ export default class CategorySectionLink {
   }
 
   get route() {
+    if (this.#linkToNew && this.activeCountable) {
+      if (get(this, this.activeCountable.propertyName) > 0) {
+        return "discovery.newCategory";
+      } else {
+        return "discovery.category";
+      }
+    }
```

Comment:
Instead of having this extra logic here, I think we can just remove "totalUnread" and "totalNew" from the countables.

Reply:
I'm not sure I understand this, can you elaborate please?

Reply:
When "new new" is enabled, the [defaults](https://github.com/discourse/discourse/blob/36ad653fa9e2a79b94f01ec573cc724455a412a4/app/assets/javascripts/discourse/app/lib/sidebar/user/categories-section/category-section-link.js#L11-L34) here do no actually make sense anymore. Therefore, if "new new" is enabled, we can just remove those "countables" type such that you no longer need this block of code.

Reply:
Ah, I see, thanks! I've made this change in https://github.com/discourse/discourse/pull/20488/commits/aa6a7d42fcc0f2893622fb790ef729e2c9ca6b0a